examples/nlp/token_classification/conf/punctuation_capitalization_config.yaml

```diff
@@ -39,50 +41,88 @@ exp_manager:
   create_checkpoint_callback: true  # Whether you want exp_manager to create a model checkpoint callback
 
 model:
-  punct_label_ids: null # will be filled during training
-  capit_label_ids: null # will be filled during training
   class_labels:
     punct_labels_file: punct_label_ids.csv
     capit_labels_file: capit_label_ids.csv
-  dataset:
-    data_dir: ??? # /path/to/training/data
-    # parameters for dataset preprocessing
-    max_seq_length: 128
```

Comment:
could you add back max_seq_length

Reply:
It is now in `train_ds`, `validation_ds`, `test_ds`. I added it back these sections